pkgs/development/python-modules/bitsandbytes/default.nix

```diff
@@ -1,39 +1,43 @@
 {
   lib,
+  torch,
+  symlinkJoin,
   buildPythonPackage,
   fetchFromGitHub,
-  python,
-  pythonOlder,
+  cmake,
   setuptools,
-  wheel,
-  torch,
   scipy,
-  symlinkJoin,
 }:
 
 let
   pname = "bitsandbytes";
-  version = "0.43.1";
+  version = "0.44.1";
 
-  inherit (torch) cudaCapabilities cudaPackages cudaSupport;
-  inherit (cudaPackages) backendStdenv cudaVersion;
+  inherit (torch) cudaPackages cudaSupport;
+  inherit (cudaPackages) cudaVersion;
+
+  cudaVersionString = lib.replaceStrings [ "." ] [ "" ] (lib.versions.majorMinor cudaVersion);
```

Comment:
This will do for an interim solution but we'll be relying on upstream not changing the naming scheme. For now let's just add a `NOTE:` warning maybe, that when it breaks it's easier to find out what needs to be done

Reply:
Done, I have just added a note.